### PR TITLE
Healthcare 12281 this visit skeleton

### PR DESCRIPTION
**Pull request by mdewey**
## Description
Adds the "this visit" page to the Healthcare team forms. This adds 3 questions to the form as a start. This is just the ground work for more features around the 3 questions. 

**NOTE**: This also has a new flag on in the forms system for array questions, call `doNotScrollOnAdd`. This need feature is a test to see how a different flow would work. Its not needed for this PR and if. its too much review, It can be pulled out into a separate PR

## Testing done
- [x] United all new components
- [x] Added e2e tests

## Screenshots
Screen Shot 2020-09-15 at 3 35 34 PM

## Definition of done
- [x] Ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/12281
- [x] form questions are displayed and updatable .

### CODE REVIEW

File: src/platform/forms-system/src/js/fields/ArrayField.jsx [hunk 1]
```diff
@@ -105,17 +105,19 @@ export default class ArrayField extends React.Component {
...
-        `table_${id}`,
-        window.Forms?.scroll || {
-          duration: 500,
-          delay: 0,
-          smooth: true,
-          offset: 0,
-        },
-      );
-    }, 100);
+    if (!this.props.uiSchema['ui:options'].doNotScrollOnAdd) {
```
Comment (rianfowler) [inline src/platform/forms-system/src/js/fields/ArrayField.jsx:108]: Let's move this to a separate PR so we can get the rest of the app approved. Thanks

**Review by rianfowler — CHANGES_REQUESTED** [review]

File: src/platform/forms-system/src/js/fields/ArrayField.jsx (hunk 1 above)
Comment (mdewey) [reply src/platform/forms-system/src/js/fields/ArrayField.jsx]: Sure thing!

**Review by massrb — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]
